adapters/nobid/usersync.go

```diff
@@ -0,0 +1,12 @@
+package nobid
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewNoBidSyncer(temp *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("nobid", 0, temp, adapters.SyncTypeRedirect)
```

Comment:
According to the GVL (https://vendorlist.consensu.org/v2/vendor-list.json), it looks like NoBid should be id 816.

Reply:
Thank you for finding this bug. It's fixed now. 
This is our first PBS Adapter and we really appreciate your recommendations and feedback.